docs/api/apiv3/endpoints/time_entries.apib

```diff
@@ -495,7 +507,729 @@ There is no form and schema resource for time entries, yet.
                 }
             }
 
+## Time entry schema [/api/v3/time_entries/schemas]
```

Comment:
Typo: `schemas` has to be `schema`

Reply:
fixed, thanks for catching. I also removed now outdated statements regarding the non existence of form and schema end points.